utils/blogPosts.js

```
@@ -0,0 +1,44 @@
+const fs = require("fs");
```

ESLint

[error] 'require' is not defined.

(no-undef)